src/util/heap/layout/mmapper.rs

```diff
@@ -49,7 +50,13 @@ pub trait Mmapper: Sync {
     // NOTE: There is a monotonicity assumption so that only updates require lock
```

Comment:
Need to update the doc comment to mention the new argument

Reply:
Done.